NEWS

```diff
@@ -2,6 +2,13 @@
 
 Sandia OpenSHMEM NEWS -- history of user-visible changes.
 
+v1.5.0rc1
+---------
+- Renamed all OpenSHMEM v1.5 interfaces from "shmemx" to "shmem" and moved all unit tests accordingly
+- Added support for UCX Transport
+- Added shmem_malloc_with_hints(though no hints supported yet)
```

Comment:
please add a space before the parens

Reply:
Is it better to re-write this? 
`- Added shmem_malloc_with_hints. Currently, no hint values are supported.`